server/container/util/src/main/java/org/apache/james/util/ReactorUtils.java

```diff
@@ -39,6 +42,13 @@
         return Mono.fromRunnable(runnable).then(Mono.empty());
     }
 
+    public static <T> BiConsumer<Optional<T>, SynchronousSink<T>> publishIfPresent() {
+        return (element, sink) -> element.ifPresent(sink::next);
+    }
+
+    public static <T, U> BiConsumer<U, SynchronousSink<T>> transformAndPublishIfNotNull(Function<U, T> mapper) {
```

Comment:
If I understand correctly, we have a helper method to promote use of nullable values?
Should we?
Or make sure people are actually providing well-behaving methods instead?

Reply:
You are right, let's remove it.